packages/devops/configs/nginx-template/servers.template.conf

```diff
@@ -570,7 +570,12 @@ server {
   listen       [::]:__PORT__;
   server_name  __DOMAIN_PREFIX____DOMAIN__;
 
+  # Nginx caches the ip address of the default frontend, which resolves to the ip address of the Elastic Load Balancer when hosting on AWS
+  # Since this ip address can change, we need to make sure we refetch the cached ip address regularly to avoid pointing to the wrong ip address
+  resolver 127.0.0.53 valid=30s; # DNS address of 127.0.0.53 is used by systemd-resolved
```

Comment:
@edmofro It turns out you do need to explicitly define the dns with a resolver here. I feel slightly uncomfortable doing this since I don't think this is guaranteed to work when deploying on premise. But perhaps we can cross that hurdle when we get to it?

The `127.0.0.53` is what's used by `systemd-resolved` which gets installed by default with `systemd`. So I think it should always be set to this on Ubuntu, but tbh this is getting rapidly outside my understanding of linux and networking infrastructure :sweat_smile:

Reply:
@passcod can I pull you in on this one?

Background is that nginx proxy_pass was playing up when forwarding traffic to a load balancer, because the load balancer IP changed and nginx cached the old IP value (apparently it resolves it once and then uses the IP after that). Rohan is setting a 30s validity period here so that the cache will be invalidated regularly

Reply:
Yeah, that's fine